define/securityHelpers.go

```diff
@@ -34,46 +25,48 @@ func LAOCreatedIsValid(data DataCreateLAO, message Message) error {
 	hash := sha256.Sum256(str)
 	//hash64 := b64.StdEncoding.EncodeToString(hash[:])
 
-	if !bytes.Equal([]byte(data.ID), hash[:]) {
+	if create && !bytes.Equal([]byte(data.ID), hash[:]) {
 		//if(hash64 != data.ID) {
 		fmt.Printf("sec3 \n")
 		fmt.Printf("%v, %v", hash, data.ID)
-		return ErrInvalidResource
+		return false
 	}
 
-	return nil
+	//TODO any more checks to perform ?
+
+	return true
 }
 
-func MeetingCreatedIsValid(data DataCreateMeeting, message Message) error {
+func MeetingCreatedIsValid(data DataCreateMeeting, message Message) bool {
 	//the last modified timestamp is equal to the creation timestamp,
-	if data.Creation != data.Last_modified {
-		return ErrInvalidResource
-	}
+	/*if data.Creation != data.Last_modified {
+		return false
+	}*/
 	//the timestamp is reasonably recent with respect to the server’s clock,
-	if data.Creation > time.Now().Unix() || data.Creation-time.Now().Unix() > MaxTimeBetweenLAOCreationAndPublish {
-		return ErrInvalidResource
+	if data.Creation > time.Now().Unix() || data.Creation-time.Now().Unix() > MaxPropagationDelay {
```

Comment:
this logic seems wrong. Let me rewrite an equivalent statement for you:

`if data.Creation > time.Now().Unix() || data.Creation > time.Now().Unix() + MaxPropagationDelay`

there's also no margin on the 'lower bound' on the data.Creation. 
Ideally you want both an upper and a lower bound with some margin, to take into account for the hosts having slightly different clocks.

Reply:
oh, yeah you're right. So we changed the logic to that :
`data.Creation < time.Now().Unix() - MaxClockDifference || data.Creation > time.Now().Unix()+MaxPropagationDelay`
This takes into account clock "dissynchronisation" for min time and for the propagation delay for max time. Maybe to be clearer we should add the MaxClockDifference to the max time as well, so we don't expect the propagation to account for propagation and Clock Dissynchronisation. What do you think ?

Reply:
Up to you, both options are reasonable.
Adding MaxClockDifference on both sides makes it easier to 'formally' define the specifications :)